components/identities/identitySidebar.tsx

```
@@ -0,0 +1,41 @@
+                ? styles.identityBoxSidebarItemActive
+                : styles.identityBoxSidebarItem
+            }
+            key={idx}
+          >
+            {item}
+          </p>
+        ))}
+      </div>
+      <div className={styles.addIdentityContainer}>
+        <PlusIcon width="17" color={theme.palette.secondary.main} />
+        <p onClick={() => {}} className={styles.addIdentity}>
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)